realsense2_camera/src/dynamic_params.cpp

```diff
@@ -103,7 +103,7 @@ namespace realsense2_camera
         try
         {
             ROS_DEBUG_STREAM("setParam::Setting parameter: " << param_name);
-#if defined(GALACTIC) || defined(ROLLING)
```

Comment:
What do you think about reversing this condition? sounds like a good idea?

Reply:
done